com.microsoft.mrtk.core/Subsystems/Speech/SpeechRecognitionSubsystemConfig.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.MixedReality.Toolkit.Subsystems
+{
+    /// <summary>
+    /// The base configuration object for all SpeechRecognitionSubsystems
+    /// </summary>
+    public class SpeechRecognitionSubsystemConfig : BaseSubsystemConfig
+    {
+
```

Comment:
You should not need base configs if you don't have any configuration data; you should be able to reuse the MRTK base subsystem config. Correct me if I'm wrong.

Reply:
I opened the PR before seeing your message on this. Will fix